samples/connectedhomeip/lock/src/util/ThreadUtil.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#pragma once
```

Comment:
I'm not sure about using `#pragma once` in sdk-nrf. I'm not saying it's wrong, but I think that it is not used in any other place, so some assumptions on this could be made.

Reply:
I'm not sure either. Can't find any hints on that in Zephyr/Linux Coding Style guides which NCS coding style is based on. I guess some NCS code owners will decide when we push the changes to main repo.